research/picai/fl_nnunet/start_server.py

```diff
@@ -73,11 +73,6 @@ def main(config: dict, server_address: str) -> None:
         # Of course nnunet stores their pytorch models differently.
         params = ndarrays_to_parameters([val.cpu().numpy() for _, val in model["network_weights"].items()])
     else:
-        # raise Exception(
```

Comment:
Did they end up fixing this on their end?

Reply:
They haven't even commented on the issue yet. Likely because I tagged it as a feature request. Maybe should have tagged it as a bug but can't change it now. I've added a comment with the issue link just to keep us up to date

Reply:
Gotcha...a comment is good!